### DISCUSSION TITLE
Variantof Cyborgworm inherits incorrect sprites

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#14822] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

![20241014162220](https://github.com/user-attachments/assets/3e0908dd-36c3-49f5-ad45-6deac0b05712)

### Reproduction steps

[Cyborgworm_m.zip](https://github.com/user-attachments/files/17361040/Cyborgworm_m.zip)

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.5.9.2 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Rokvach** [comment]
Fixed and tested the fix to be working correctly, fix coming in the next update. Closing discussion.
